### PR TITLE
Update learning pack - general maintenance

### PR DESCRIPTION
**Pull request by Enet4**
- Update project dependencies
- Minor configuration adjustments
- Update site footer to say 2023 instead of 2021
- Adjust note styles so that they are all in the same file and the headers are less heavy
- Update general text styles so that the font weight is not so thin to the point of becoming barely readable in some devices.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <footer class="inner">
     <p class="copyright">
-        © 2021 <a href="https://bioinformatics.ua.pt">UA.PT Bioinformatics</a>, maintained by the <a href="https://github.com/bioinformatics-ua/dicoogle#project-committers">Dicoogle team</a>
+        © 2022 <a href="https://bioinformatics.ua.pt">UA.PT Bioinformatics</a>, maintained by the <a href="https://github.com/bioinformatics-ua/dicoogle#project-committers">Dicoogle team</a>
```
Comment (Enet4) [inline _includes/footer.html:3]:
```suggestion
        © 2023 <a href="https://bioinformatics.ua.pt">UA.PT Bioinformatics</a>, maintained by the <a href="https://github.com/bioinformatics-ua/dicoogle#project-committers">Dicoogle team</a>
```

**Comment by bastiao** [comment]
Any thoughts on the message below? 

```
bundle install

[!] There was an error parsing `Gemfile`: No such file or directory @ rb_sysopen - https://pages.github.com/versions.json. Bundler cannot continue.

 #  from /mnt/c/Users/bastiao/Projects/dicoogle-learning-pack/Gemfile:10
 #  -------------------------------------------
 #  # uncomment the line below. To upgrade, run `bundle update github-pages`.
 >  versions = JSON.parse(open('https://pages.github.com/versions.json').read)
 #  gem "github-pages", versions['github-pages'], group: :jekyll_plugins
```

**Review by rlebre — COMMENTED** [review]
Having the same issue here:

> [!] There was an error parsing `Gemfile`: No such file or directory @ rb_sysopen - https://pages.github.com/versions.json. Bundler cannot continue

**Comment by Enet4** [comment]
It's been a year. :) Let's try again.

**Review by rlebre — APPROVED** [review]
Works! Thanks! :shipit:

**Pull request merged**